public/scripts/extensions/vectors/settings.html

```diff
@@ -196,152 +196,153 @@ <h4>
                     </label>
                     <textarea id="vectors_file_template_db" class="margin0 text_pole textarea_compact" rows="3" placeholder="Use &lcub;&lcub;text&rcub;&rcub; macro to specify the position of retrieved text."></textarea>
-                    <label for="vectors_file_position_db">Injection Position</label>
+                    <label for="vectors_file_position_db" data-i18n="Injection Position">Injection Position</label>
                     <div class="radio_group">
                         <label>
                             <input type="radio" name="vectors_file_position_db" value="2" />
-                            <span>Before Main Prompt / Story String</span>
+                            <span data-i18n="Before Main Prompt / Story String">Before Main Prompt / Story String</span>
                         </label>
                         <!--Keep these as 0 and 1 to interface with the setExtensionPrompt function-->
                         <label>
                             <input type="radio" name="vectors_file_position_db" value="0" />
-                            <span>After Main Prompt / Story String</span>
+                            <span data-i18n="After Main Prompt / Story String">After Main Prompt / Story String</span>
                         </label>
                         <label for="vectors_file_depth_db" title="How many messages before the current end of the chat." data-i18n="[title]How many messages before the current end of the chat.">
                             <input type="radio" name="vectors_file_position_db" value="1" />
-                            <span>In-chat @ Depth</span>
+                            <span data-i18n="In-chat @ Depth">In-chat @ Depth</span>
                             <input id="vectors_file_depth_db" class="text_pole widthUnset" type="number" min="0" max="999" />
                             <span>as</span>
                             <select id="vectors_file_depth_role_db"  class="text_pole widthNatural">
-                                <option value="0">System</option>
-                                <option value="1">User</option>
-                                <option value="2">Assistant</option>
+                                <option value="0" data-i18n="System">System</option>
+                                <option value="1" data-i18n="User">User</option>
+                                <option value="2" data-i18n="Assistant">Assistant</option>
                             </select>
                         </label>
                     </div>
                 </div>
                 <div class="flex-container">
                     <div id="vectors_files_vectorize_all" class="menu_button menu_button_icon" title="Vectorize all files in the Data Bank and current chat.">
-                        Vectorize All
+                        <div data-i18n="Vectorize All">Vectorize All</div>
```

Comment:
Replace div with span

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7